### DISCUSSION TITLE
Welcome to AralTools!

### DISCUSSION THREAD

**Discussion by Yivan000** [tudlang/araltools#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by clavzno** [comment]
genuinely love this project, thank you!!! excited to see what you have in store next for this :^)

**Reply by Yivan000** [reply]
Thank you! There'll be more exciting new features coming soon :)))

**Comment by arvj9** [comment]
I'm so ecstatic seeing this project coming to fruition.

Reminds me of the time I wanted to make one but then gave up immediately when I couldn't quite figure out how to grab the Course Offerings information. At the time, I didn't bother continuing since there was only like a week to enlist, and my skillset wasn't up to par to make something on my own while dealing with my own unrelated classes.

**Reply by Yivan000** [reply]
Thank you! 

*If you're curious, the parser for the course offerings is in [this file](https://github.com/tudlang/araltools/blob/main/lib/araltools/skedmaker/parser.dart) .*
